pandas/tseries/resample.py

```diff
@@ -652,9 +652,6 @@ def _convert_obj(self, obj):
             # Cannot have multiple of periods, convert to timestamp
             self.kind = 'timestamp'
 
-        if not len(obj):
-            self.kind = 'timestamp'
-
```

Comment:
I'm not sure what the goal of this was. It made [this test](https://github.com/pydata/pandas/pull/12771/files#diff-f89a17cef9382e5fafdc58604f1744c0R2258) I added fail